### PR TITLE
Feature/add kyc modes

### PR DESCRIPTION
**Pull request by ali-ayyad-1**
Added Partner conducted KYC guide and removed complete headless KYC guide.

Ticket Link: https://www.notion.so/immersve/Create-Partner-Conducted-KYC-page-3eabf72a661144f68d0da8c963c9fbdf?pvs=4

### CODE REVIEW

File: imsv-docs-astro/src/content/docs/guides/kyc-modes/partner-conducted-kyc.mdoc [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    }'
+    ```
+    {% /code %}
+
+
+- Poll [spending prerequisites](https://docs.immersve.com/api-reference/get-spending-prerequisites/) until KYC Check is not returned in the response.
+
+{% code title="Spending prerequisite endpoint" %}
+```bash
+curl --location 'https://test.immersve.com/api/spending-prerequisites' \
```
Comment (ncjones) [inline imsv-docs-astro/src/content/docs/guides/kyc-modes/partner-conducted-kyc.mdoc:101]:
```suggestion
curl "https://test.immersve.com/api/spending-prerequisites" \
```

File: imsv-docs-astro/src/content/docs/guides/kyc-modes/partner-conducted-kyc.mdoc [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+-H 'Content-Type: application/json' \
+-H "X-Api-Key: $api_key" \
+-H "X-Api-Secret: $api_secret" \
+-H "X-Account-Id: $cardholder_account_id" \
```
Comment (ncjones) [inline imsv-docs-astro/src/content/docs/guides/kyc-modes/partner-conducted-kyc.mdoc:105]:
```suggestion
-H "X-Account-Id: ${cardholder_account_id}" \
```

File: imsv-docs-astro/src/content/docs/guides/kyc-modes/partner-conducted-kyc.mdoc [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    ```
+    {% /code %}
+
+
+- Poll [spending prerequisites](https://docs.immersve.com/api-reference/get-spending-prerequisites/) until KYC Check is not returned in the response.
+
+{% code title="Spending prerequisite endpoint" %}
+```bash
+curl --location 'https://test.immersve.com/api/spending-prerequisites' \
+-H 'Content-Type: application/json' \
```
Comment (ncjones) [inline imsv-docs-astro/src/content/docs/guides/kyc-modes/partner-conducted-kyc.mdoc:102]:
```suggestion
-H "Content-Type: application/json" \
```

File: imsv-docs-astro/src/content/docs/guides/kyc-modes/partner-conducted-kyc.mdoc [hunk 4]
```diff
@@ -0,0 +1,138 @@
+---
+title: Partner Conducted KYC
+slug: guides/kyc-modes/partner-conducted-kyc
```
Comment (ncjones) [inline imsv-docs-astro/src/content/docs/guides/kyc-modes/partner-conducted-kyc.mdoc:3]:
```suggestion
slug: guides/partner-conducted-kyc
```

File: imsv-docs-astro/src/content/docs/guides/kyc-modes/partner-conducted-kyc.mdoc [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+    card_program_id="<your_card_program_id>"
+    cardholder_account_id="<cardholder_account_id>"
+    funding_source_id="<funding_source_id>"
+    ```
+{% /code %}
+
+- [Submit KYC statement](https://docs.immersve.com/api-reference/submit-partner-kyc-statement#submit-partner-kyc-statement) about the cardholder account.  Provide ‘passall’ as the middle name, the check should be successful regardless of the validity of the rest of the details.
+{% code title="Submit KYC a statement" %}
+    ```bash
+    curl --location --request PUT "https://test.immersve.com/api/accounts/${cardholder_account_id}/partner-kyc-statement" \
```
Comment (ncjones) [inline imsv-docs-astro/src/content/docs/guides/kyc-modes/partner-conducted-kyc.mdoc:42]:
```suggestion
    curl -X PUT "https://test.immersve.com/api/accounts/${cardholder_account_id}/partner-kyc-statement" \
```

File: imsv-docs-astro/src/content/docs/guides/kyc-modes/partner-conducted-kyc.mdoc [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+    --header 'Content-Type: application/json' \
+    --header 'Accept: application/json' \
+    --header "x-api-key: ${kyc_api_key}" \
+    --header "x-api-secret: ${kyc_api_secret}" \
```
Comment (ncjones) [inline imsv-docs-astro/src/content/docs/guides/kyc-modes/partner-conducted-kyc.mdoc:46]:
```suggestion
    -H"x-api-secret: ${kyc_api_secret}" \
```

File: imsv-docs-astro/src/content/docs/guides/kyc-modes/partner-conducted-kyc.mdoc [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+--data '{
+  "cardProgramId": "'${card_program_id}'",
+  "fundingSourceId": "'${funding_source_id}'",
+  "spendableAmount": 100,
+  "spendableCurrency": "USD",
+  "kycType": "partner-conducted",
+  "kycRegion":"NZL"
+}'
+
+  # the response should contain:
```
Comment (ncjones) [inline imsv-docs-astro/src/content/docs/guides/kyc-modes/partner-conducted-kyc.mdoc:115]: Pull this to a separate code block

File: imsv-docs-astro/src/content/docs/guides/kyc-modes/partner-conducted-kyc.mdoc [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+  # 		},
+  # 		...
+  #   ]
+  # }
+```
+{% /code %}
+
+Calling the "get spending prerequisites" endpoints queues a KYC check. When the
+KYC check is queued, a KYC profile for that account is created with a 'created'
+status. The status changes to 'pending' when the check is started and the KYC
```
Comment (ncjones) [inline imsv-docs-astro/src/content/docs/guides/kyc-modes/partner-conducted-kyc.mdoc:133]: Use double quotes or backticks.

File: imsv-docs-astro/src/content/docs/guides/kyc-modes/partner-conducted-kyc.mdoc [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+subject to an agreement between Immersve and the partner. Ask Immersve support
+for more details.
+{% /note %}
+
+See the [KYC API endpoint references](https://docs.immersve.com/api-reference/kyc/).
+
+- Set up additional environment variables
+
+{% code title="Environment variable setup" %}
+    ```bash
```
Comment (braden-holgate) [inline imsv-docs-astro/src/content/docs/guides/kyc-modes/partner-conducted-kyc.mdoc:25]: Do we wan this indent? We need a convention.

Comment (ali-ayyad-1) [reply imsv-docs-astro/src/content/docs/guides/kyc-modes/partner-conducted-kyc.mdoc:25]: I think it fits the code block better

File: imsv-docs-astro/src/redirects.yml [hunk 10]
```diff
@@ -35,6 +35,7 @@
 /guides/custodial wallets/card-lifecycle: "/guides/card-lifecycle"
 /guides/custodial wallets/physical-cards: "/guides/issue-a-virtual-card"
 /guides/custodial-wallets: "/use-cases/custodial-wallets"
+/guides/complete-headless-kyc-checks: "guides/partner-conducted-kyc"
```
Comment (ncjones) [inline imsv-docs-astro/src/redirects.yml:38]: please sort the lines

Comment (ali-ayyad-1) [reply imsv-docs-astro/src/redirects.yml:38]: What do you mean by that ?

Comment (ali-ayyad-1) [reply imsv-docs-astro/src/redirects.yml]: I have changed the order to have it sorted alphabetically, I guess this is what you meant by sort the lines. @ncjones

**Review by braden-holgate — APPROVED** [review]

**Review by braden-holgate — APPROVED** [review]

**Pull request merged**